<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfo</class>
 <widget class="QWidget" name="FileInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileInfo</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>属性：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="isDirCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>目录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isExecutableCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isFileCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>符号链</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isHiddenCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>隐藏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isReadableCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isSymLinkCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isWritableCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>文件大小：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>创建时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>最后修改时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>最后访问时间：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="fileNameLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileSizeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileCreateLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileModifyLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileReadLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
